code-of-conduct.md

```diff
@@ -1,3 +1,37 @@
 # OpenTelemetry Community Code of Conduct
 
-OpenTelemetry follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
\ No newline at end of file
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+We are bound by the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and
+covered by that in more serious violations and prefer the reporting of
+violations to be made to us as outlined below.
+
+## Reporting violations
+
+If you have seen or experienced unacceptable behavior or anything that would
+make our community less welcoming, please speak up! 
+
+The Governance Committee is the right place to bring your concerns and can be
+reached via cncf-opentelemetry-governance@lists.cncf.io. You can also get in
+touch with any committee member if you feel safer this way: any one of us can
```

Comment:
I think it may be a good idea to provide a more specific example and be more direct about this. I'd take a look at the CNCF's CoC reporting procedure for inspiration https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md#submit-in-writing

We may want to just take their wording almost exactly substituting our GC for their CoC Committee

> If you do not want your report to be received by all members of the CoC Committee, either because you want to submit a report anonymously or because one of the CoC Committee members has a conflict of interest, you may send your report directly to any individual member

Reply:
That doc is great. I reorganized the final part of our code of conduct with parts from that. Thank you for the hint! Can you please take another look?